helmfile.d/helmfile-60.teams.yaml

```diff
@@ -187,7 +186,17 @@ releases:
           - azure{{ end }}
           {{- if $team | get "stack.sitespeed" false }}
           - sitespeed{{ end }}
-
   {{- end }}
   {{- end }}
+  {{- if hasKey $team "jobs" }}
+  - name: jobs-{{ $teamId }}
+    installed: {{ has $v.cluster.id $team.clusters }}
```

Comment:
You are missing an essential insight here:

Jobs, like services, are ONLY defined for ONE cluster scope, so they run only there. Don't deviate from that approach. You now publish jobs to ALL clusters the team operates on.

Reply:
I would say add your job snippet in the loop that deals with the team services.

Reply:
> Jobs, like services, are ONLY defined for ONE cluster scope, so they run only there. Don't deviate from that approach. You now publish jobs to ALL clusters the team operates on.

Forgive me if I'm wrong, I don't fully understand everything about this syntax yet. Of course I don't want publish jobs to all clusters. But I don't think that's necessarily true with this condition: 
`{{- if hasKey $team "jobs" }}`

> I would say add your job snippet in the loop that deals with the team services.

Based on your suggestion, `jobs` will only be available if there is a `services` key, which is not necessarily true, correct? Although I reckon the `services` key will always be used.

Reply:
If jobs are defined only in cluster file then it should be fine. E.g.:
```
env/clouds/aws/demo/jobs.demo.yaml
```
```
env/clouds/aws/dev/jobs.demo.yaml
```

Reply:
Of course, I am too much out of the game it seems ;p